src/rgw/rgw_common.h

```diff
@@ -726,6 +726,7 @@ struct rgw_bucket {
   std::string oid; /*
                     * runtime in-memory only info. If not empty, points to the bucket instance object
                     */
+  std::string name_space;
```

Comment:
can we drop the `_` and use `namespace` to match the librados `set_namespace()` api?